src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -368,13 +377,13 @@ limitations under the License.
 
   <clusterExtension code="0x0300">
     <attribute side="server" code="0x4000" define="COLOR_CONTROL_ENHANCED_CURRENT_HUE" type="int16u" min="0x0000" max="0xFFFF" writable="false" default="0x0000" optional="true">EnhancedCurrentHue</attribute>
-    <attribute side="server" code="0x4001" define="COLOR_CONTROL_ENHANCED_COLOR_MODE" type="enum8" min="0x00" max="0xFF" writable="false" default="0x01">EnhancedColorMode</attribute>
+    <attribute side="server" code="0x4001" define="COLOR_CONTROL_ENHANCED_COLOR_MODE" type="EnhancedColorModeEnum" min="0x00" max="0xFF" writable="false" default="0x01">EnhancedColorMode</attribute>
     <attribute side="server" code="0x4002" define="COLOR_CONTROL_COLOR_LOOP_ACTIVE" type="int8u" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">ColorLoopActive</attribute>
-    <attribute side="server" code="0x4003" define="COLOR_CONTROL_COLOR_LOOP_DIRECTION" type="int8u" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">ColorLoopDirection</attribute>
+    <attribute side="server" code="0x4003" define="COLOR_CONTROL_COLOR_LOOP_DIRECTION" type="ColorLoopDirectionEnum" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">ColorLoopDirection</attribute>
```

Comment:
Just a note: this seems to not match the spec?  Spec says "uint8" with a "max 1" constraint....

If we think the spec should say `ColorLoopDirectionEnum`, then the spec needs to be updated accordingly, yes?  And the test plan (which just says to check that the value is a uint8, without even checking for the "max 1" constraint).

And if the spec is still in flux here, we should, imo, not touch this XML until the spec stabilizes.

@jmartinez-silabs @ReneJosefsen

Reply:
Good catch, I need to do some research on this to verify if this has always been the case or if this is a mistake on the spec side.

Reply:
Looks like the editorial https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7808 changed this to max 1 🤔 

I can see the PR does state: "Adjusted constraints of ColorLoopDirection and ColorLoopActive" so it seems intentional.

Reply:
A bit more concerning that COLOR_CONTROL_COLOR_LOOP_ACTIVE and COLOR_CONTROL_COLOR_LOOP_DIRECTION have been marked as int8 but the spec has since 1.0 been uint8?

Reply:
"int8u" is what the spec calls "uint8", so that part is all fine.

Reply:
Oh yeah, that was on me, read it as int8t. Oops. All good.

Reply:
@bzbarsky-apple I believe the Spec PR was finally merged. There is a new PR from Rene for some small bits that were still discussed. Since a Enum is used now. Do we still have to define a min and Max? Is using kUnknownEnumValue, in SDK,  to ensure proper limits enough?

Test plan should also be updated if we proceed with this pr.

Reply:
The spec does not use an enum for ColorLoopDirection right now. We can certainly change that, but currently the ColorLoopDirection attribute is still marked as uint8 in the attribute table.

Reply:
@ReneJosefsen  Thanks, ColorLoopDirection attribute changed to uint8.